src/components/WorldMap/WorldMap.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from "react";
+import BingMaps from "ol/source/BingMaps";
+import Map from "ol/Map.js";
+import TileLayer from "ol/layer/Tile.js";
+import View from "ol/View.js";
+import classes from "./WordMap.module.css";
+
+const styles = [
+  "RoadOnDemand",
+  "Aerial",
+  "AerialWithLabelsOnDemand",
+  "CanvasDark",
+  "OrdnanceSurvey",
+];
+const layers = [];
+
+const initMap = () => {
+  layers.push(
+    new TileLayer({
+      visible: true,
+      preload: Infinity,
+      source: new BingMaps({
+        key: "Ahuf0ANrUzpA6kcYt7ZiYIIu5DR9oF6RG62SautIoD2UKxAmbxhPn--scNKOS6Vm",
```

Comment:
We can see a way to handle the secret key so that they are hidden and not available.